dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/acl/TrackerAccessManager.java

```diff
@@ -47,6 +47,8 @@
 public interface TrackerAccessManager {
   List<String> canRead(UserDetails user, TrackedEntity trackedEntity);
 
+  List<String> canRead(UserDetails user, TrackedEntity trackedEntity, boolean skipOwnershipCheck);
```

Comment:
When would I use this one? You said some method was temporary, is it this one? if so mark it deprecated for removal (add the version or issue in which it will be removed)